INFO - 2025-01-07 17:47:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-07 17:47:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-07 17:48:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-01-07 17:48:49 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:50 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:50 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:52 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:52 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:52 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:52 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-01-07 17:48:53 --> Error connecting to the database: mysqli_sql_exception: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'admin', Object(SensitiveParameterValue), 'test', 3306, '', 0)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı in D:\xampp\htdocs\codeigniter\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\codeigniter\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\codeigniter\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\codeigniter\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\codeigniter\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 D:\xampp\htdocs\codeigniter\app\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
#6 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(933): App\Controllers\AuthController->login()
#7 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#8 D:\xampp\htdocs\codeigniter\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\codeigniter\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 D:\xampp\htdocs\codeigniter\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 D:\xampp\htdocs\codeigniter\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-01-07 17:48:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Hedef makine etkin olarak reddettiğinden bağlantı kurulamadı
[Method: POST, Route: login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `email` = :email:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\AuthController.php(42): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-07 18:50:03 --> Error: Class "App\Controllers\CategoryModel" not found
[Method: GET, Route: /]
in APPPATH\Controllers\HomeController.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\HomeController->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-01-07 18:50:19 --> Error: Class "App\Controllers\CategoryModel" not found
[Method: GET, Route: /]
in APPPATH\Controllers\HomeController.php on line 9.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\HomeController->index()
 2 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
 3 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
INFO - 2025-01-07 19:00:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-07 19:00:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-07 19:01:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2025-01-07 19:01:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
